src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/VmrSyncAdditionalMappingsTest.cs

```diff
@@ -62,14 +62,14 @@ protected override async Task CopyReposForCurrentTest()
         File.WriteAllText(
             _filePath,
             "A file that needs to be copied outside of the src folder");
-        File.WriteAllText(_specialRepoPath / "eng" / _versionDetailsName, Constants.EmptyVersionDetails);
+        File.WriteAllText(_specialRepoPath / "eng" / Constants.VersionDetailsName, Constants.EmptyVersionDetails);
```

Comment:
There's a class called `VersionFiles` and this should come from there.

Reply:
Check this commit - https://github.com/dotnet/arcade-services/pull/2154/commits/6cc30bb8610788c0bed30a8e96188bfc2c3570d6
Something similar should this be done in this PR